content/500-platform/550-platfrom-cli/about.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: 'About'
```

Comment:
I'm missing a section about the intstallation of the CLI or at least a link to the actual installation docs?